src/components/Exchange/SwapBox.js

```diff
@@ -2050,7 +2050,7 @@ export default function SwapBox(props) {
                 <span className="muted">Leverage slider</span>
               </ToggleSwitch>
               {isLeverageSliderEnabled && (
-                <LeverageSlider isLong={isLong} leverageOption={leverageOption} onChange={setLeverageOption} />
+                <LeverageSlider isPositive={isLong} value={leverageOption} onChange={setLeverageOption} />
```

Comment:
i'm afraid we have two LeverageSlider components now, maybe incorect merging

This one accept IsLong and leverageOption
[src/components/Exchange/LeverageSlider.tsx](https://github.com/gmx-io/gmx-interface/blob/b6c6d0f38d114c870f30aac00e5557a69dbe1e13/src/components/Exchange/LeverageSlider.tsx#L41)

also we have src/components/LeverageSlider.tsx

Let's remove (1)

Reply:
done it by myself